MIGRATING_v2_to_v3.md

```diff
@@ -0,0 +1,236 @@
+
+This is a newly created interface with methods listed below.
+- `TokenAuthorizationServerPolicyRuleAction getToken()`
+- `AuthorizationServerPolicyRuleActions setToken(TokenAuthorizationServerPolicyRuleAction token)`
+
+### Package `com.okta.sdk.resource.authorization.server.policy.AuthorizationServerPolicyRuleConditions`
+
+This is a newly created interface with methods listed below.
+- `ClientPolicyCondition getClients()`
+- `GrantTypePolicyRuleCondition getGrantTypes()`
+- `PolicyPeopleCondition getPeople()`
+- `OAuth2ScopesMediationPolicyRuleCondition getScopes()`
+- `AuthorizationServerPolicyRuleConditions setClients(ClientPolicyCondition clients)`
+- `AuthorizationServerPolicyRuleConditions setGrantTypes(GrantTypePolicyRuleCondition grantTypes)`
+- `AuthorizationServerPolicyRuleConditions setPeople(PolicyPeopleCondition people)`
+- `AuthorizationServerPolicyRuleConditions setScopes(OAuth2ScopesMediationPolicyRuleCondition scopes)`
+   
+### Package `com.okta.sdk.resource.authorization.server.policy.TokenAuthorizationServerPolicyRuleAction`
+
+This is a newly created interface with methods listed below.
+- `Integer getAccessTokenLifetimeMinutes()`
+- `Integer getRefreshTokenLifetimeMinutes()`
+- `Integer getRefreshTokenWindowMinutes()`
+- `TokenAuthorizationServerPolicyRuleAction setAccessTokenLifetimeMinutes(Integer accessTokenLifetimeMinutes)`
+- `TokenAuthorizationServerPolicyRuleAction setRefreshTokenLifetimeMinutes(Integer refreshTokenLifetimeMinutes)`
+- `TokenAuthorizationServerPolicyRuleAction setRefreshTokenWindowMinutes(Integer refreshTokenWindowMinutes)`
+
+### Package `com.okta.sdk.resource.identity.provider.IdentityProvider`
+
+Below method has been removed.
+- `deleteSigningKey(String keyId)`
+
+### Package `com.okta.sdk.resource.policy.PolicyList` 
+
+The Interface has been removed.
+The Interface `com.okta.sdk.resource.authorization.server.AuthorizationServerPolicyList` should be used instead.
+
+### Package `com.okta.sdk.resource.user.factor.CustomHotpUserFactor`
+
+This is a newly created interface with methods listed below.
+- `String getFactorProfileId()`
+- `CustomHotpUserFactorProfile getProfile()`
+- `CustomHotpUserFactor setFactorProfileId(String factorProfileId)`
+- `CustomHotpUserFactor setProfile(CustomHotpUserFactorProfile profile)`
+
+### Package `com.okta.sdk.resource.user.factor.CustomHotpUserFactorProfile`
+
+This is a newly created interface with methods listed below.
+- `String getSharedSecret()`
+- `CustomHotpUserFactorProfile setSharedSecret(String sharedSecret)`
+
+### Package `com.okta.sdk.resource.user.factor.VerifyUserFactorResponse`
+
+Below method has undergone a signature change.
+- `getExpiresAt()` return type changed from `String` to `Date`
+   - The property's `expiresAt` type contains a date-time string, so it's convenient to use the `Date` type
+
+Below method has been removed.
+- `setExpiresAt(String expiresAt)`
```

Comment:
```suggestion
Below method has undergone a signature change.
- `getExpiresAt()` return type changed from `String` to `Date`
   - The property's `expiresAt` type contains a date-time string, so it's convenient to use the `Date` type
```

The "removed" is implied in the previous section

Reply:
Fixed